recipe/meta.yaml

```diff
@@ -1,34 +1,41 @@
-{% set name = "typeguard" %}
 {% set version = "4.1.5" %}
+{% set min_python = "3.8" %}
+{% set min_typing_extensions = "4.7.0" %}
```

Comment:
this validates that the upstream `typing_extensions` pin hasn't changed

Reply:
How does this add any validation? And the variable is only used once?